### PR TITLE
[Doc] Added Workflow integration documentation

### PR DESCRIPTION
**Pull request by yann-eugone**
As a complement to #5028 I created documentation to explain workflow integration using the third party library I created.

### CODE REVIEW

File: docs/cookbook/recipe_workflow_integration.rst [hunk 1]
```diff
@@ -0,0 +1,100 @@
+Integrate Symfony Workflow Component
+====================================
+
+If you are using Symfony `Workflow Component`_ and if you wish to integrate some features to Sonata,
```
Comment (kunicmarko20) [inline docs/cookbook/recipe_workflow_integration.rst]: `and if you wish to use it with Sonata` sounds better to me, wdyt?

File: docs/cookbook/recipe_workflow_integration.rst [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+there is a 3rd party library that provide toolkit classes.
+
+You can find it on `Packagist`_ and `GitHub`_.
+
+
```
Comment (kunicmarko20) [inline docs/cookbook/recipe_workflow_integration.rst]: Can you remove the additional line?

File: docs/cookbook/recipe_workflow_integration.rst [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+Downloading the library
+-----------------------
+
+From a terminal, use composer to install the library :
+
+.. code-block:: bash
+
+    $ composer require yokai/sonata-workflow
+
+
```
Comment (kunicmarko20) [inline docs/cookbook/recipe_workflow_integration.rst]: Can you remove the additional line?

File: docs/cookbook/recipe_workflow_integration.rst [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+Usage
+-----
+
+Let's say we have a ``BlogPost`` entity that is under a Symfony Workflow :
```
Comment (kunicmarko20) [inline docs/cookbook/recipe_workflow_integration.rst]: `Workflow:`

File: docs/cookbook/recipe_workflow_integration.rst [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+                     publish: fa fa-times
+
+   sonata_admin:
+       extensions:
+           admin.extension.blog_post_workflow:
+               admins:
+                   - admin.blog_post
+
+
+You are all set. If you visite your admin page in edit or show mode,
```
Comment (kunicmarko20) [inline docs/cookbook/recipe_workflow_integration.rst]: `visit`

File: docs/cookbook/recipe_workflow_integration.rst [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+you will see something like this.
```
Comment (kunicmarko20) [inline docs/cookbook/recipe_workflow_integration.rst]: `like this:`

File: docs/cookbook/recipe_workflow_integration.rst (hunk 4 above)
Comment (OskarStark) [reply docs/cookbook/recipe_workflow_integration.rst]: IMO workflow should be lowercase

File: docs/cookbook/recipe_workflow_integration.rst [hunk 7]
```diff
@@ -0,0 +1,98 @@
+Integrate Symfony Workflow Component
+====================================
+
+If you are using Symfony `Workflow Component`_ and if you wish to use it with Sonata,
+there is a 3rd party library that provide toolkit classes.
```
Comment (kunicmarko20) [inline docs/cookbook/recipe_workflow_integration.rst]: provides.

Comment (kunicmarko20) [reply docs/cookbook/recipe_workflow_integration.rst]: missed one, sorry

**Comment by greg0ire** [comment]
cc @lyrixx

File: docs/cookbook/recipe_workflow_integration.rst [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+You can use the provided extension to take care of your entity admin.
+
+.. code-block:: yaml
+
+   # config/packages/sonata_admin.yml
+   services:
+       admin.blog_post:
+           class: App\Admin\BlogPostAdmin
+           public: true
+           arguments: [~, App\Entity\PullRequest, 'Yokai\SonataWorkflow\Controller\WorkflowController']
```
Comment (greg0ire) [inline docs/cookbook/recipe_workflow_integration.rst]: Please use multi-line style here

**Comment by greg0ire** [comment]
Thanks @yann-eugone !

**Comment by jordisala1991** [comment]
Does this mean we can close #5028 ?

**Comment by kunicmarko20** [comment]
Yes 🎉

**Comment by yann-eugone** [comment]
Thanks :)

**Comment by kunicmarko20** [comment]
Thank you! great addition 😄

**Pull request merged**